app/index.html

```diff
@@ -9,3 +9,4 @@
 {% include sections/30_speakers.html %}
 {% include sections/35_speakers-cta.html %}
 {% include sections/60_dont-miss-it-cta.html %}
+{% include sections/40_tickets.html %}
```

Comment:
I guess numbering the files wasn't explicit enough for you :bowtie: 
this should come before 60_dont-miss-it-cta